src/public/curriculum/mothed/mothed-proj-content.json

```diff
@@ -0,0 +1,272 @@
+{
+  "code": "moth",
```

Comment:
Do you want to be able to load this curriculum and the `mothed.json` curriculum? The `code` value needs to be different from each other to be able to load the two different curriculums. Currently, it's loading this one, and not the `mothed.json` curriculum.

Reply:
I'm the one that added this second mothed-proj-content file just to keep around the file pre-merge. I didn't know that it would mess up the loading. 

Probably we should just remove this file from the PR to reduce any confusion.

Reply:
Never mind. I forgot you also have to add the curriculum into the app-config.json file to actually load it. So it is loading the `mothed.json` file. There may be value to keeping to `mothed-proj-content.json` as it shows the curriculum without the Workspaces tab in nav tabs, and without the problem document. We could add this to the app-config.json ```    "moth-proj-content": {
      "content": "curriculum/mothed/mothed-proj-content.json"
    },```
    And change the `code` value in the `mothed-proj-content.json` to still be able to demo it.